django/test/testcases.py

```diff
@@ -1286,6 +1286,8 @@ class LiveServerTestCase(TransactionTestCase):
     """
     host = 'localhost'
     static_handler = _StaticFilesHandler
+    # This makes it easier to change in a subclass.
```

Comment:
Looks good. I think we can skip the comment and also order the attributes alphabetically (i.e. put this above `static_handler`.